README.md

```diff
@@ -1,29 +1,32 @@
-[![Build Status](https://travis-ci.org/clemahieu/raiblocks.svg?branch=master)](https://travis-ci.org/clemahieu/raiblocks) 
-___
-
+RaiBlocks  
+====
+[![Build Status](https://travis-ci.org/clemahieu/raiblocks.svg?branch=master)](https://travis-ci.org/clemahieu/raiblocks)
 ![Logo of RaiBlocks](https://github.com/clemahieu/raiblocks/blob/master/logo.png)
 
 # What is RaiBlocks?
 RaiBlocks is designed to be a feeless, instant, high throughput cryptocurrency.
 
-We've applied the philosophy of "Do one thing and do it well" giving you performance and scalability unmatched by any other platform.
-
-___
+We've applied the philosophy of "do one thing and do it well", giving you performance and scalability unmatched by any other platform.
 
 ### Key Features:
 * RaiBlocks utilizes a novel block-lattice architecture, unlike conventional blockchains used in many other cryptocurrencies.
 * The network requires minimal resources, no high-power mining hardware, and can process high transaction throughput.
 * Offers feeless, instantaneous transactions, as well as unlimited scalability, making RaiBlocks ideal for peer-to-peer transactions.
 * As of December 2017, the RaiBlocks network has processed over four million transactions with an unpruned ledger size of only 1.7GB.
 
-***
+For more information, see [RaiBlocks.net](https://raiblocks.net/) or read the [RaiBlocks whitepaper](https://raiblocks.net/media/RaiBlocks_Whitepaper__English.pdf). 
 
-### For more information:
-- [Homepage](https://raiblocks.net)
+### Resources
+- [RaiBlocks website](https://raiblocks.net)
 - [Whitepaper](https://raiblocks.net/media/RaiBlocks_Whitepaper__English.pdf)
 - [Roadmap](https://raiblocks.net/media/raiblocks-roadmap-v2-en.png)
-
-***
+- [Discord chat](https://chat.raiblocks.net)
+- [RaiBlocks forum](https://forum.raiblocks.net/)
+- [GitHub wiki](https://github.com/clemahieu/raiblocks/wiki)
 
 ### Build instructions: 
 https://github.com/clemahieu/raiblocks/wiki/Build-Instructions
+
+### Need help?
+- See the documentation at the [RaiBlocks wiki](https://github.com/clemahieu/raiblocks/wiki) for help and more information.
+- Ask for help in the [Discord #development](https://discordapp.com/channels/370266023905198083/370285507185344524) channel.
```

Comment:
Add the contributor's guide from the wiki

Reply:
Done 👍